package.json

```diff
@@ -66,6 +66,9 @@
     "worker-loader": "^1.1.0",
     "xxhashjs": "^0.2.1"
   },
+  "resolutions": {
```

Comment:
Could you try updating acorn-dynamic-import to 3.0.0 instead?
It depends on acorn@5.

I believe resolutions is a yarn only feature. We have to avoid using it.